src/services/asterPlotConverterService.ts

```diff
@@ -132,24 +132,23 @@ export class AsterPlotConverterService {
         this.highlightedDataPoints = [];
     }
 
-    // tslint:disable-next-line: function-name
-    public static isDataValid(categorical: AsterPlotColumns<DataViewCategoryColumn & DataViewValueColumn[] & DataViewValueColumns>): boolean {
+    public static isDataValid(categorical: { Category: powerbi.DataViewCategoryColumn; Y: powerbi.DataViewValueColumn[]; }): boolean {
         return categorical
             && categorical.Category
             && !isEmpty(categorical.Category.values)
             && !isEmpty(categorical.Y)
             && !isEmpty(categorical.Y[0].values);
     }
 
-    private containsHighlights(categorical: AsterPlotColumns<DataViewCategoryColumn & DataViewValueColumn[] & DataViewValueColumns>): boolean {
+    private containsHighlights(categorical: { Category: powerbi.DataViewCategoryColumn; Y: powerbi.DataViewValueColumn[]; }): boolean {
         return categorical && categorical.Y && categorical.Y[0] && !!(categorical.Y[0].highlights);
```

Comment:
Lets make this return simpler:
`return !!(categorical?.Y?.[0]?.highlights);`

Reply:
Done.